backend/danswer/background/run_all.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
what is this for?

Reply:
https://medium.com/@codingmaths/bin-bash-what-exactly-is-this-95fc8db817bf